contracts/Ethleverage.sol

```diff
@@ -106,55 +121,64 @@ contract Ethleverage {
 
   function leverage(uint256 _ethMarketPrice, uint256 _priceFloor) payable public returns (bool sufficient) {
     //TO-DO: w/ price floor or leverage ratio, determine the number of layers and LR
-    uint256  calcCR = (_ethMarketPrice.mul(makerLR)).div(_priceFloor);
-
+    uint256  calcCR = (_ethMarketPrice.mul(makerLR)).div(_priceFloor.mul(100));
 
+    //investor assignments
     Investor memory sender;
     sender = investors[msg.sender];
     investorAddresses.push(msg.sender);
-
     sender.layers = layers;
     sender.prinContr = msg.value;
```

Comment:
We could probably think of a better variable name for this, I don't remember what prinContr is

Reply:
haha! I forgot to finish the variable name. It should read principalContribution